kubernetes/templates/mysql/statefulset.yaml

```diff
@@ -34,27 +44,43 @@ spec:
           mountPath: /mnt/config-map
       containers:
       - name: mysql
-        image: {{ .Values.mysql.image.repository }}:{{ .Values.mysql.image.tag }}
+        image: "{{ .Values.mysql.image.repository }}:{{ .Values.mysql.image.tag }}"
         env:
+        - name: NODE_ENV
+          value: "production"
         - name: MYSQL_DATABASE
-          value: kubevious
-        - name: MYSQL_ALLOW_EMPTY_PASSWORD
-          value: "1"
+          value: {{ .Values.mysql.db_name }}
+        - name: MYSQL_USER
+          value: {{ .Values.mysql.db_user }}
+        - name: MYSQL_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              {{- if .Values.mysql.existingSecret }}
+              name: {{ .Values.mysql.existingSecret }}
```

Comment:
A question. Is this really needed? My understanding is that ppl would deploy Kubevious in its own namespace. In that case there wouldn't be any other secret where credentials are stored. Wouldn't forcing the secret name help with reducing number of "variables" ?

Reply:
this is just in case that anyone want to have an external mysql running... so they can use an existing secret with that info

Reply:
that was my question actually. if they deploy kubevious in a different namespace, they wouldn't be able to access the other secret anyways

Reply:
i've removed the option of an existing secret until we move the mysql to a dependency.